script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -141,12 +142,12 @@ Judging by her lack of reply, I guess she really is alone...
 [sha:03f3f8cc8e880f8c1e04acc93896ab408a51fcf9]{
 -- Page 481, Offset 6887.
 -- まあ、それはこっちも助かるけど―――
-Well, I guess it's better for me that way―――
+Well, I guess that makes things easier for me, at least―――
```

Comment:
Might revert this just because this TL makes it sound like he has an agenda with her